### ISSUE TITLE
Multiple traits

### ISSUE THREAD

**Issue by ryanmasondavies** [iotize/Braille#2]
This was omitted for reasons of time restriction, but BRLAccessibilityElementFinder should be made capable of finding elements based on multiple traits provided to it.
